dev/archery/README.md

```diff
@@ -48,4 +48,90 @@ Additionally, if you would prefer to install everything at once,
 `pip install -e "arrow/dev/archery[all]"` is an alias for all of
 the above subpackages.
 
-For some prior art on benchmarking in Arrow, see [this prototype](https://github.com/apache/arrow/tree/0409498819332fc479f8df38babe3426d707fb9e/dev/benchmarking).
\ No newline at end of file
+For some prior art on benchmarking in Arrow, see [this prototype](https://github.com/apache/arrow/tree/0409498819332fc479f8df38babe3426d707fb9e/dev/benchmarking).
+
+# Usage
+## Integration tests
```

Comment:
We already have docs for the Integration tests in https://github.com/apache/arrow/blob/main/docs/source/format/Integration.rst, I suggest augmenting this document rather than the README.